app/routers/quizzes.py

```diff
@@ -7,8 +7,9 @@
 router = APIRouter(prefix="/quiz", tags=["Quiz"])
 
 
-@router.post("/", response_model=QuizResponse)
+@router.post("/")
 async def create_quiz(quiz: Quiz):
+    """Returns the ID of created quiz in a dictionary with key as 'quiz_id'"""
```

Comment:
Some feedback on docstrings
- Here, the _meat_ of the method is not mentioned. The purpose of this method is not just to _return the id of the created quiz_. That is a side effect or post effect. The main task this method is doing is actually _creating the quiz_. 
- No need to mention things like `dictionary` and `with key as `quiz_id`. If you use a ResponseModel, anyone will be able to see what are the keys and what are their types that will be returned from this method call. `dictionary` is specific to python that's why you should avoid it. Plus someone reading the code, developer or not developer, should be able to understand what a method does just from the docstring